src/components/App/App.tsx

```diff
@@ -30,7 +36,28 @@ class App extends React.Component<{}, AllData> {
 
   componentDidMount = () => {
     discoverPlanets()
+      .then(response => {
+        if (!response.ok) {
+          throw new Error();
```

Comment:
I've actually never used `throw new Error()` I'm curious what this actually does? Should we create a simple error component?

Reply:
It creates a new `Error` object that will get picked up by whatever's in the `.catch` block. And yes, we totally could create a simple error component!